ui/pages/remote-mode/setup/setup-daily-allowance/remote-mode-setup-daily-allowance.component.tsx

```diff
@@ -169,27 +171,39 @@ export default function RemoteModeSetupDailyAllowance() {
       return;
     }
 
-    const newAllowance = {
-      tokenType: selectedAllowanceToken,
+    const selectedAsset = assetsWithBalance.find(
+      (asset) => asset.address === selectedAllowanceAddress,
+    ) as TokenInfo;
+
+    // TODO: handle error
```

Comment:
Low-ish priority but could be worth capturing these TODO's in a ticket to make sure they don't get missed downstream.